File: porting-console-app-rhill34/visibility/Employee.cs
using System;
namespace visibility
{
    public class Employee
    {
        public Employee(string fName, string lName, Address address, Position position)
        {
            FName = fName;
            LName = lName;
            Address = address;
            Position = position;
        }

        protected string FName { get; set; }
        protected string LName { get; set; }

        //Creates the Employee Full Name
        public string FullName
        {
            get
            {
                string x = FName + ' ' + LName;
                return x;
            }
        }

        public Address Address { get; set; }
        public Position Position { get; set; }

        public override string ToString()
        {
            return "Employee: " + Environment.NewLine +
                "Position: " + Position + Environment.NewLine +
                "Full Name: " + FullName + Environment.NewLine +
                 Address;
        }
    }
}

File: porting-console-app-rhill34/javaFiles/visibility/Employee.java
package visibility;

public class Employee {
    protected String fName;
    protected String lName; //package
    private Address address;
    private Position position;
    public Employee(String fName, String lName, Address address, Position position) {
        super();
        this.fName = fName;
        this.lName = lName;
        this.address = address;
        this.position = position;
    }

    public String getFullName() {
        return fName + " " + lName;
    }

    public Position getPosition() {
        return position;
    }
    @Override
    public String toString() {
        return "Employee "+ this.getFullName() + "\n"+ address + "\nPosition=" + position;

    }


}
